### ISSUE TITLE
Update documentation around kubernetes installation

### ISSUE THREAD

**Issue by spencergilbert** [vectordotdev/vector#9073]
@binarylogic I think I'd prefer to tackle and plan a larger change to the install docs (moving kubectl/kustomize under manual), if that's alright with you I'll merge this and open an additional issue.

_Originally posted by @spencergilbert in https://github.com/timberio/vector/issues/9024#issuecomment-912693637_

**Comment by jszwedko** [comment]
@spencergilbert is this still relevant? If so, could you provide more details on what the proposed "larger change" is? If it is just "moving kubectl/kustomize under manual" than could we reword this to make that clear that is the extent of the proposed changes here?

**Comment by spencergilbert** [comment]
😅 not particularly - it seems like I wanted to reorganize the instructions more fully, but this can probably be closed given our current priorities and expectations around the doc site.
